src/main/java/games/strategy/triplea/settings/SettingsWindow.java

```diff
@@ -77,76 +90,103 @@ private static JComponent createContents(final Runnable closeListener) {
         .collect(Collectors.toList());
   }
 
-  private static JComponent buildTab(final List<ClientSettingSwingUiBinding> settings, final Runnable closeListener) {
-    return JPanelBuilder.builder()
-        .borderLayout()
-        .addCenter(tabMainContents(settings))
-        .addSouth(buttonPanel(settings, closeListener))
+  private static JComponent buildTabPanel(final Iterable<ClientSettingSwingUiBinding> settings) {
+    final JPanel panel = JPanelBuilder.builder()
+        .borderEmpty(10)
         .build();
-  }
-
-  private static JComponent tabMainContents(final Iterable<ClientSettingSwingUiBinding> settings) {
-    final JPanelBuilder contents = JPanelBuilder.builder()
-        .gridBagLayout(3);
-
-    // Add settings, one per row, columns of 3:
-    // setting title (JLabel) | input component (eg: radio buttons) | description (JTextArea)}
-
-    settings.forEach(setting -> {
-      contents.add(JPanelBuilder.builder()
-          .horizontalBoxLayout()
-          .add(
-              JLabelBuilder.builder()
-                  .text(setting.title)
-                  .leftAlign()
-                  .maximumSize(200, 50)
+    panel.setLayout(new GridBagLayout());
```

Comment:
I'm manually building the `GridBagLayout` in this method instead of using the swinglib `GridBagHelper` stuff that's part of `JPanelBuilder`.  The swinglib code didn't provide enough fidelity to generate the specific layout required in this window.  `GridBagHelper` is used for numerous other UIs, and I didn't want to be concerned with possibly breaking those layouts by extending the functionality of `GridBagHelper` in this PR.